Schedule 친구 초대 기능

cafitac opened the pull request:
ScheduleSerializer의 fields 변경
ScheduleViewSet Create 뷰 Response 변경
ScheduleViewSet (invite, join) action 추가 - fcm 메시지 기능 추가 예정

Commit:
Merge branch 'user-feature' into member-feature

Commit:
Merge branch 'member-feature' into develop

hanseongd commented on scheduler/member/serializers.py:
```diff
@@ -17,6 +29,7 @@ class Meta:
 
     def create(self, validated_data):
         user_data = validated_data.pop('user')
+        user_data = user_data
```
불필요한 부분

hanseongd commented on scheduler/member/views.py:
```diff
@@ -49,4 +59,87 @@ def update(self, request, *args, **kwargs):
+                'status': 200,
+                'user': user_data,
+            })
+        except:
```
정확히 어떤 에러가 발생했을 때 인지 알 수 있어야해

hanseongd commented on scheduler/scheduler/views.py:
```diff
@@ -0,0 +1,23 @@
+
+
+# Todo: SocialLoginView 따라 들어가면 get_social_login이라는 뷰가 있음. 추후 커스터마이징에 이용
+class FacebookLogin(SocialLoginView):
```
테스트 된 부분이야?

hanseongd commented on scheduler/schedule/views.py:
```diff
@@ -1,13 +1,73 @@
+            'message': '일정 생성 완료'
+        }, status=status.HTTP_201_CREATED, headers=headers)
+
+    @action(detail=True, methods=['GET'])
```
테스트 중인건 추후에 완료 후에 올리기

hanseongd commented on scheduler/schedule/views.py:
```diff
@@ -1,13 +1,73 @@
+        })
+
+    @action(detail=True, methods=['GET'])
+    def join(self, request, pk=None):
```
try문을 2단으로 가져갈 필요 없이 하나로 작성할 수 있고 지금 방식 보다 더 나은 방식이 있어보이는데 수정해보기

hanseongd reviewed (commented):
try except 문을 제대로 사용하고 있지 않아서 except쪽 모두 다시 수정하고 수정하다 보면 더 짧고 보기 좋게 수정할 수 있을거야

cliffechoi commented:
다른 브런치에서의 merge 기록이 이번에도 꼭 필요했어?

Commit:
Schedule 친구 초대 기능 추가

ADD: ScheduleSerializer 생성
ADD: ScheduleViewSet 생성 후 create 뷰 커스터마이징, 일정 초대 action 추가
